rust/xaynet-core/src/message/payload/update.rs

```diff
@@ -280,11 +279,11 @@ pub(in crate::message) mod tests_helpers {
     use std::convert::TryFrom;
 
     use super::*;
-    use crate::{
-        crypto::ByteObject,
-        mask::{object::MaskMany, seed::EncryptedMaskSeed},
-        SumParticipantPublicKey,
+    pub(in crate::message) use crate::mask::object::serialization::tests::{
+        mask_many as masked_model,
+        mask_one as masked_scalar,
```

Comment:
I think it would be better to bring in `mask_object` instead and use that in the below

Reply:
done

Reply:
thanks, why the thumbs down?

Reply:
Sorry, that was a misclick, I wanted to thumb this up actually :smile: